packages/basic/test/logger.test.js

```diff
@@ -0,0 +1,422 @@
+import { describe, test } from 'node:test'
+import { strictEqual, deepStrictEqual, throws } from 'node:assert'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { buildPinoOptions, loadFunction } from '../lib/utils.js'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
you should be able to use `import.meta.dirname` and `import.meta.filename`

Reply:
moved to packages/utils, no need to do so since it's cjs